tests/virt/node/general/test_machinetype.py

```
@@ -50,6 +50,26 @@ def vm_for_machine_type_test(request, cpu_for_migration, unprivileged_client, na
         yield vm
 
 
+@pytest.fixture()
+def explicit_machine_type(is_s390x_cluster):
+    return MachineTypesNames.s390_ccw_virtio_rhel7_6 if is_s390x_cluster else MachineTypesNames.pc_q35_rhel7_6
+
+
+@pytest.fixture()
```

Ruff (0.14.13)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)